test/conversion.test.js

```diff
@@ -27,6 +27,52 @@ describe('Curl converter should', function() {
 		});
 	});
 
```

Comment:
We can split the tests into two.
1. Checking the correct method being added from parameters.
2. Checking the priority of methods